src/components/IPFS/Dropzone.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react'
```

Comment:
We don't use Component anymore for new components. Please refactor to functional components and [hooks](https://reactjs.org/docs/hooks-reference.html)